src/Miew.js

```diff
@@ -3559,6 +3559,12 @@ Miew.prototype._initOnSettingsChanged = function () {
 
   on(['transparency', 'resolution', 'palette'], () => {
     this.rebuildAll();
+
+    const volume = this._getVolumeVisual();
```

Comment:
Please replace the code on resolution change in separate `on('resolution', ()=>{...}` and I think you don't need to set _`needRender = true` because `rebuildAll` has already done it.

Reply:
Replaced "resolution code".
But _needRender = true nonetheless is needed, because rebuildAll is called before changing the defines.